### ISSUE TITLE
Function argument comments are empty

### ISSUE THREAD

**Issue by f0rt** [MarcGrol/golangAnnotations#13]
Hey team,
I'm using `annotationParser.ParseSourceDir()` to parse a file into a model. I've noticed that `model.Field.DocLines` and `model.Field.CommentsLines` are always empty. I've done some debugging and saw that ast model is empty too.
This is a known issue: https://github.com/golang/go/issues/35484
They suggest using `ast.NewCommentMap()` to load the function argument comments: https://go.dev/play/p/zDcLxXqTk0
Thanks!

**Comment by MarcGrol** [comment]
Do you have a business driver that forces you to extract function-parameter comments?

**Comment by f0rt** [comment]
Yes, I'm using the annotations to generate a CLI. I'm copying comments to the CLI parameter comments.

**Comment by MarcGrol** [comment]
Could you re-apply your proposed change?
- without breaking existing tests and 
- without introducing any new external packages (like github.com/fOrt)

Apologies from my side. I should have given this feedback when you offered your PR
